### ISSUE TITLE
Exception occurred when make latex on getfem.git

### ISSUE THREAD

**Issue by tkoyama010** [sphinx-doc/sphinx#6046]
**Exception occurred**
Exception occurred when compile latex document in project getfem.

**To Reproduce**
```
$ git clone https://git.savannah.nongnu.org/git/getfem.git
$ cd getfem
$ git checkout devel-tetsuo-VersionUp-sphinx
$ bash autogen.sh
$ ./configure
$ cd doc/sphinx/
$ make latex
```

**Expected behavior**
Exception occurred:
  File "/usr/lib/python2.7/dist-packages/sphinx/writers/latex.py", line 706, in check_latex_elements
    logger.warning(msg % key)
TypeError: not all arguments converted during string formatting
The full traceback has been saved in /tmp/sphinx-err-6HUGqT.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!.

**Your project**
<http://getfem.org/>

**Screenshots**
None

**Environment info**
- OS: [Ubuntu18.10]
- Python version: [2.7.15]
- Sphinx version: [1.7.9]
- Sphinx extensions:  ['sphinx.ext.imgmath', 'sphinx.ext.autodoc', 'sphinx.ext.coverage', 'sphinx.ext.doctest']
- Extra tools: [docutils == 0.11 jinja2 == 2.3.1 pygments == 1.6]

**Additional context**
[sphinx-err-6HUGqT.log](https://github.com/sphinx-doc/sphinx/files/2847648/sphinx-err-6HUGqT.log)

**Comment by tkoyama010** [comment]
Same error occured also in v1.8.4.
[sphinx-err-BwoVYi.log](https://github.com/sphinx-doc/sphinx/files/2847669/sphinx-err-BwoVYi.log)

**Comment by jfbu** [comment]
I did not test  (my g++ compiler is too old for `./configure` to be happy; besides the Makefile hardcodes in `~/.local/bin` location of sphinx-build if I am not wrong,).

I noticed

```
latex_elements = {
    'preamble': 'user_preamble',
}
```

in your `conf.py`, which will lead to other problems it should be

```
latex_elements = {
    'preamble':  user_preamble,
}
```

This should not be related to your problem and it will show only at time of executing pdflatex on the produced tex files. You also have

```
\\usepackage[draft]{minted}\\fvset{breaklines=true}
```

which seems unneeded. Package minted hacks fancyvrb to add this `breaklines=true` option, but Sphinx for a long time (I think 1.4.2, May 2016) wraps long code lines. You don't need this line.

**Comment by tk0miya** [comment]
It seems your project defines invalid `latex_elements`. And it is a cause of the `TypeError`.
```
latex_elements = [
    ('preamble',user_preamble),
]
```
http://git.savannah.nongnu.org/cgit/getfem.git/tree/doc/sphinx/source/conf.py#n298

So please update your configuration.

On the other hand, the TypeError is raised from LaTeX builder. And we can fix it easily. So I'll work for it.

**Comment by tk0miya** [comment]
I just fixed about crash in #6050. It does not means to accept wrong configuration of getfem. Please update your conf.py.
Thank you for reporting.

**Comment by tkoyama010** [comment]
I've confirmed. Thank you for fixing.
